<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1092</width>
    <height>645</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>7680</width>
    <height>4320</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Inventory</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="1">
     <widget class="QStackedWidget" name="mainWindowStackedWidget">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <widget class="QWidget" name="pointOfSalePage">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <layout class="QGridLayout" name="pointOfSalePageLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QWidget" name="point_of_sale_window_widget" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0">
           <property name="sizeConstraint">
            <enum>QLayout::SetNoConstraint</enum>
           </property>
           <item>
            <layout class="QHBoxLayout" name="main_header_layout">
             <item>
              <widget class="QPushButton" name="push_btn_2">
               <property name="text">
                <string>Dashboard</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="push_btn">
               <property name="text">
                <string>Inventory</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="point_of_sale_hero_layout">
             <item>
              <layout class="QVBoxLayout" name="LeftPOSCol">
               <property name="sizeConstraint">
                <enum>QLayout::SetMaximumSize</enum>
               </property>
               <property name="leftMargin">
                <number>10</number>
               </property>
               <property name="rightMargin">
                <number>10</number>
               </property>
               <item>
                <widget class="QWidget" name="widget" native="true">
                 <property name="enabled">
                  <bool>true</bool>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout">
                  <item>
                   <layout class="QHBoxLayout" name="customerSelectDiv">
                    <item>
                     <widget class="QComboBox" name="customerSelect">
                      <property name="minimumContentsLength">
                       <number>1</number>
                      </property>
                      <property name="placeholderText">
                       <string>Select Customer</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QPushButton" name="addCustomerBtn">
                      <property name="maximumSize">
                       <size>
                        <width>30</width>
                        <height>30</height>
                       </size>
                      </property>
                      <property name="font">
                       <font>
                        <pointsize>18</pointsize>
                        <bold>true</bold>
                       </font>
                      </property>
                      <property name="text">
                       <string>+</string>
                      </property>
                      <property name="iconSize">
                       <size>
                        <width>20</width>
                        <height>20</height>
                       </size>
                      </property>
                      <property name="shortcut">
                       <string>Ctrl+Z</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <layout class="QHBoxLayout" name="productSearchLeftDiv">
                    <item>
                     <widget class="QComboBox" name="productSearchLeft">
                      <property name="placeholderText">
                       <string>Enter Barcode/ID of Product</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QPushButton" name="pushButton">
                      <property name="maximumSize">
                       <size>
                        <width>30</width>
                        <height>30</height>
                       </size>
                      </property>
                      <property name="font">
                       <font>
                        <pointsize>13</pointsize>
                       </font>
                      </property>
                      <property name="text">
                       <string>âœ”</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <widget class="QTableWidget" name="itemCartTable"/>
                  </item>
                  <item>
                   <layout class="QHBoxLayout" name="priceSectionDiv">
                    <item>
                     <layout class="QVBoxLayout" name="priceLeftColDiv">
                      <item>
                       <layout class="QHBoxLayout" name="horizontalLayout_5">
                        <item>
                         <widget class="QLabel" name="totalItemsLabel">
                          <property name="text">
                           <string>Total Item(s) :</string>
                          </property>
                          <property name="alignment">
                           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                          </property>
                         </widget>
                        </item>
                        <item>
                         <widget class="QLabel" name="totalItemsCountLabel">
                          <property name="text">
                           <string>0</string>
                          </property>
                         </widget>
                        </item>
                       </layout>
                      </item>
                      <item>
                       <layout class="QHBoxLayout" name="horizontalLayout_7">
                        <item>
                         <widget class="QLabel" name="discountLabel">
                          <property name="text">
                           <string>Discount (Rs): </string>
                          </property>
                         </widget>
                        </item>
                        <item>
                         <widget class="QLineEdit" name="discountInput"/>
                        </item>
                       </layout>
                      </item>
                      <item>
                       <layout class="QHBoxLayout" name="horizontalLayout_9">
                        <item>
                         <widget class="QLabel" name="cashReturnLabel">
                          <property name="text">
                           <string>Cash Return :</string>
                          </property>
                         </widget>
                        </item>
                        <item>
                         <widget class="QLabel" name="cashReturnCountLabel">
                          <property name="text">
                           <string>Rs 0</string>
                          </property>
                         </widget>
                        </item>
                       </layout>
                      </item>
                     </layout>
                    </item>
                    <item>
                     <layout class="QVBoxLayout" name="priceRightColDiv">
                      <item>
                       <layout class="QHBoxLayout" name="horizontalLayout_6">
                        <item>
                         <widget class="QLabel" name="priceLabel">
                          <property name="text">
                           <string>Price :</string>
                          </property>
                         </widget>
                        </item>
                        <item>
                         <widget class="QLabel" name="priceCountLabel">
                          <property name="text">
                           <string>Rs 0</string>
                          </property>
                         </widget>
                        </item>
                       </layout>
                      </item>
                      <item>
                       <layout class="QHBoxLayout" name="horizontalLayout_8">
                        <item>
                         <widget class="QLabel" name="cashPaidLabel">
                          <property name="text">
                           <string>Cash Paid :</string>
                          </property>
                         </widget>
                        </item>
                        <item>
                         <widget class="QLineEdit" name="cashPaidInput"/>
                        </item>
                       </layout>
                      </item>
                      <item>
                       <layout class="QHBoxLayout" name="horizontalLayout_10">
                        <item>
                         <widget class="QLabel" name="netPriceLabel">
                          <property name="text">
                           <string>Net Price :</string>
                          </property>
                         </widget>
                        </item>
                        <item>
                         <widget class="QLabel" name="netPriceCountLabel">
                          <property name="font">
                           <font>
                            <pointsize>11</pointsize>
                            <bold>true</bold>
                           </font>
                          </property>
                          <property name="text">
                           <string>Rs 0</string>
                          </property>
                         </widget>
                        </item>
                       </layout>
                      </item>
                     </layout>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_2">
                    <item>
                     <widget class="QPushButton" name="clearBtn">
                      <property name="text">
                       <string>Clear</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QPushButton" name="printBtn">
                      <property name="text">
                       <string>Print</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QPushButton" name="payBtn">
                      <property name="text">
                       <string>Pay</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="rightPOSCol">
               <item>
                <layout class="QHBoxLayout" name="productSearchRightDiv">
                 <item>
                  <widget class="QLineEdit" name="productSearchRightInput"/>
                 </item>
                 <item>
                  <widget class="QPushButton" name="productSearchRightBtn">
                   <property name="text">
                    <string>Search</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="product_page">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <layout class="QGridLayout" name="product_layout"/>
      </widget>
      <widget class="QWidget" name="dashboard_page">
       <layout class="QGridLayout" name="dashboard_layout"/>
      </widget>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QWidget" name="main_menu_widget" native="true">
      <layout class="QVBoxLayout" name="main_menu_layout">
       <item>
        <widget class="QPushButton" name="menu_home_btn">
         <property name="text">
          <string>Home</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="menu_dashboard_btn">
         <property name="text">
          <string>Dashboard</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="menu_product_btn">
         <property name="text">
          <string>Product</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1092</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
